frontend/src/tests/nonprofitApply/nonprofitApplyImage.test.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
```

Comment:
(nit) can change name to criteriaImage.test.js

Reply:
done